pages/careers/_open-positions.html

```diff
@@ -1,24 +1,59 @@
-    <h3>Senior Full-Stack Engineer</h3>
+    <p>
+      We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue
+      supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity,
+      we would love to hear from you. Here are the positions we're currently looking for:</p>
+    <h3>Software Engineer</h3>
     <h4>What You'll Work On</h4>
     <ul>
-      <li>Engineering a fundamentally better DevOps experience.</li>
+      <li>
+        <a href="/houston">Gruntwork Houston</a>: build a fundamentally better DevOps experience.
+      </li>
+      <li>
+        <a href="/infrastructure-as-code-library">Infrastructure as Code Library</a>: create reusable infrastructure
+        modules for a variety of infrastructure (e.g., Kubernetes, ELK, Consul, Vault, Kafka, MongoDB, InfluxDB,
+        Jenkins, etc), using a variety of tools (e.g., Terraform, Go, Python, Bash, Docker, Packer, etc), across many
+        clouds (e.g., AWS, GCP, and Azure).
+      </li>
+      <li>
+        <a href="https://github.com/gruntwork-io/" target="_blank">Open Source</a>: contribute to our open source
+        projects, including our <a href="/infrastructure-as-code-library/#open-source">open source modules</a>,
+        <a href="https://github.com/gruntwork-io/terragrunt" target="_blank">Terragrunt</a>,
+        <a href="https://github.com/gruntwork-io/terratest" target="_blank">Terratest</a>,
+        <a href="https://github.com/gruntwork-io/cloud-nuke" target="_blank">cloud-nuke</a>,
+        <a href="https://github.com/gruntwork-io/bash-commons" target="_blank">bash-commons</a>, and more.
+      </li>
     </ul>
     <h4>What You'll Do</h4>
     <ul>
-      <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
+      <li>
+        <strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on
+        Serverless apps in Node.js, write Terraform modules, write Python scripts and Bash scripts, do frontend
+        development, learn the newest tech from AWS, GCP, and Azure, invent new ways of managing infrastructure.
+        We're looking for full-stack engineers who can go deep in any area. We're less concerned with prior experience
```

Comment:
This part still mentions `full-stack`, consider changing to `software`
```suggestion
        We're looking for software engineers who can go deep in any area. We're less concerned with prior experience
```